src/mito2/src/manifest/manager.rs

```diff
@@ -222,10 +241,11 @@ impl RegionManifestManagerInner {
         // recover from storage
         // construct manifest builder
         let mut version = MIN_VERSION;
-        let last_checkpoint = store.load_last_checkpoint().await?;
-        let checkpoint = last_checkpoint
-            .map(|(_, raw_checkpoint)| RegionCheckpoint::decode(&raw_checkpoint))
-            .transpose()?;
+        let checkpoint = Self::last_checkpoint(&store).await?;
+        let last_checkpoint_version = checkpoint
+            .as_ref()
+            .map(|checkpoint| checkpoint.last_version)
+            .unwrap_or(MIN_VERSION);
         let mut manifest_builder = if let Some(checkpoint) = checkpoint {
             info!(
                 "Recover region manifest {} from checkpoint version {}",
```

Comment:
I think we should print the manifest path in the logs in this file.

Reply:
Which path are you referring to? the manifest path is already there